docs/tutorial.rst

```diff
@@ -910,6 +910,10 @@ By default, the following variables are available for use:
   * ``user``: The username of the current user
   * ``hostname``: The name of the host computer
   * ``cwd``: The current working directory
+  * ``dir_cwd``: The dirname of the current working directory, e.g. ``/path/to`` in
```

Comment:
A bit :).  Feel free to PR in a better name.

On Thu, Aug 13, 2015 at 4:42 PM Aaron Meurer <notifications@github.com>
wrote:

> In docs/tutorial.rst
> <https://github.com/scopatz/xonsh/pull/346#discussion_r37022742>:
>
> > @@ -910,6 +910,10 @@ By default, the following variables are available for use:
> >    * ``user``: The username of the current user
> >    * ``hostname``: The name of the host computer
> >    * ``cwd``: The current working directory
> > +  * ``dir_cwd``: The dirname of the current working directory, e.g. ``/path/to`` in
>
> Yeah, isn't "cwd_dir" kind of like "ATM machine"?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/scopatz/xonsh/pull/346/files#r37022742>.
>